cmd/juju/status/status_test.go

```diff
@@ -171,8 +171,9 @@ var (
 			"current": "started",
 			"since":   "01 Apr 15 01:23+10:00",
 		},
-		"dns-name":    "controller-0.dns",
-		"instance-id": "controller-0",
+		"dns-name":     "controller-0.dns",
+		"ip-addresses": []string{"controller-0.dns"},
```

Comment:
Fair point, they are the same strings used in the rest of this test suite in place of address strings, but they can be changed.